canopy/app/src/loadSaplings.js

```diff
@@ -48,6 +49,21 @@ async function loadCurrentSapling(userSaplingsResponse) {
   return false;
 }
 
+async function loadSaplingStyles(saplingResponse) {
+  const saplingStyleFiles = saplingResponse
+    .map(s => s.styleFiles)
```

Comment:
Could you give this and the following arguments a better name? Its very difficult to figure out what we're mapping over or flatMapping

Reply:
updated